### PR TITLE
XRDDEV-1864 CS localisation and notifications update

### PR DESCRIPTION
**Pull request by Riippi**
https://jira.niis.org/browse/XRDDEV-1864

- Remove excessive vuex actions for notifications. Now similar to Security Server.
- Sort the localisations file alphabetically
- Small localisation fixes

### CODE REVIEW

File: src/centralserver/admin-ui/src/views/AppLogin.vue [hunk 1]
```diff
@@ -193,10 +193,7 @@ export default (
                 this.loading = false;
               })
               .catch((error) => {
```
Comment (teemukin65) [inline src/centralserver/admin-ui/src/views/AppLogin.vue:195]: this catch is for errors from the `then` -pipeline starting after `fetchUserData()`

File: src/centralserver/admin-ui/src/views/AppLogin.vue [hunk 2]
```diff
@@ -219,18 +216,15 @@ export default (
               });
```
Comment (teemukin65) [inline src/centralserver/admin-ui/src/views/AppLogin.vue:216]: This error handler starting from line 200 is handling rejected promise of `fetchUserData()`  i.e. the reject handler in` then(`

**Review by teemukin65 — CHANGES_REQUESTED** [review]

File: src/centralserver/admin-ui/src/views/AppLogin.vue [hunk 3]
```diff
@@ -219,18 +216,15 @@ export default (
...
             }
             this.$store.dispatch(
-              StoreTypes.actions.SHOW_ERROR_MESSAGE_CODE,
-              'login.generalError',
+              StoreTypes.actions.SHOW_ERROR_MESSAGE,
```
Comment (teemukin65) [inline src/centralserver/admin-ui/src/views/AppLogin.vue:219]: Here we can be pretty sure that error is of type AxiosError , ie. we could use the .SHOW_ERROR, too.

Comment (Riippi) [reply src/centralserver/admin-ui/src/views/AppLogin.vue:219]: Fixed this

File: src/centralserver/admin-ui/src/ui-types.ts [hunk 4]
```diff
@@ -55,18 +56,16 @@ export interface Notification {
   timeAdded: number;
   timeout: number;
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
```
[the remark this answers is not in the document]
Comment (Riippi) [reply src/centralserver/admin-ui/src/ui-types.ts:58]: Fixed

File: src/centralserver/admin-ui/src/views/AppLogin.vue [hunk 5]
```diff
@@ -219,18 +216,15 @@ export default (
...
             // Clear loading state
             this.loading = false;
           },
         )
         .catch((error) => {
-          this.$store.dispatch(
-            StoreTypes.actions.SHOW_ERROR_MESSAGE_RAW,
-            error,
-          );
+          this.$store.dispatch(StoreTypes.actions.SHOW_ERROR, error);
```
[the remark this answers is not in the document]
Comment (Riippi) [reply src/centralserver/admin-ui/src/views/AppLogin.vue:227]: We discussed this in slack and I will create a new task for refactoring the login system here.

**Pull request merged**